lib/unhangout-routes.js

```diff
@@ -491,18 +549,51 @@ module.exports = {
             showEventOverview(req, res, "admin.ejs");
         });
 
-        app.get("/myevents/", ensureAuthenticated, function(req, res) {
+        app.get("/myevents/", ensureAuthenticated, utils.ensureAdmin,function(req, res) {
             showEventOverview(req, res, "myevents.ejs");
         });
 
+        app.post("/myevents/", ensureAuthenticated, utils.ensureAdmin, function(req, res) {
+
+            var eventID = req.body.eventId; 
+
+            var event = db.events.get(req.body.eventId);
+
+            if (!event) {
+                return res.send(400, "Missing event.");
```

Comment:
Minor thing, but the correct status code for missing document would be 404.  Though I guess you could argue that posting to a non-existent document is a "Bad Request"...